stock_available/views/product_template_view.xml

```diff
@@ -81,6 +81,9 @@
         <field name="model">product.template</field>
         <field name="inherit_id" ref="stock.product_template_kanban_stock_view" />
         <field name="arch" type="xml">
+            <xpath expr="//kanban" position="inside">
+                <field name="type" />
+            </xpath>
             <xpath expr="//div[@name='product_lst_price']" position="after">
                 <div
                     t-if="record.type.raw_value == 'product'"
```

Comment:
```suggestion
                    t-if="record.show_on_hand_qty_status_button.raw_value"
```

Reply:
I actually see this reasonable, I'll change it

Reply:
Change applied @rousseldenis